dashboard/config/routes.rb

```diff
@@ -335,7 +335,14 @@
       end
     end
 
-    resources :rubrics, only: [:edit, :new]
+    resources :rubrics, only: [:edit, :new, :create, :update]
+
+    # resources :lessons, only: [:edit, :update] do
```

Comment:
Note to self... I am just now seeing this comment in the routes file.  I'd like to see if it passes drone first and then go back to delete this.